README.md

```diff
@@ -58,7 +58,7 @@ pnpm i -D eslint-plugin-import eslint-import-resolver-typescript
 yarn add -D eslint-plugin-import eslint-import-resolver-typescript
 ```
 
-**Important when using `eslint-plugin-i` and `npm`**: Use `npm i -D eslint-plugin-import@eslint-plugin-i@latest eslint-import-resolver-typescript`, or you will end up with both `eslint-plugin-import` and `eslint-plugin-i` in your node_modules.
+**Important when using `eslint-plugin-import-x` and `npm`**: Use `npm i -D eslint-plugin-import@npm:eslint-plugin-import-x@latest eslint-import-resolver-typescript`, or you will end up with both `eslint-plugin-import` and `eslint-plugin-import-x` in your node_modules.
```

Comment:
It might be impossible to use `eslint-plugin-import-x` as an alias of `eslint-plugin-import` now. Might possibly just remove the tip.

Reply:
Nope, the settings prefix is hardcoded as `import-x`, so aliasing will not work as expected.